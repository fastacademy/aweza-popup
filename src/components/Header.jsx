import { h, Component } from 'preact'

export default class Header extends Component {
  render () {
    return (
      <div class="aweza-category">
        <b>{this.props.title}</b>
      </div>
    )
  }
}
