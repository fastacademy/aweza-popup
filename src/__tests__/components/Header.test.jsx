import preact from 'preact';
import render from 'preact-render-to-json';
import Header from "../../components/Header";

// /** @jsx preact.h */

test('component', () => {
    const tree = render(<Header title={"Lorem Ipsum"}/>);

    expect(tree).toMatchSnapshot();
});
