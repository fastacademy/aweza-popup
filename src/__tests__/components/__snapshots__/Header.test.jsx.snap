// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component 1`] = `
<div
  class="aweza-category"
>
  <b>
    Lorem Ipsum
  </b>
</div>
`;
